features/dataset-view.feature

```diff
@@ -330,6 +330,42 @@ Feature: a user visit the dataset page
 		| pre_03AUG2015_update 								| folder containing originally submitted data files, prior to update Aug 3rd 2015.	| Directory 		| 50.00 MiB 	|                 | ftp://climb.genomics.cn/pub/10.5524/101001_102000/101001/pre_03AUG2015_update |
 		| readme.txt 										|				                                                                    | Readme 			| 337 B 		|                 | ftp://climb.genomics.cn/pub/10.5524/101001_102000/101001/readme.txt |
 
+	@ok @javascript @pr521
+	Scenario: To see the citation box and and the citation source
+		Given I am not logged in to Gigadb web site
+		And I am on "/dataset/101001"
+		And I should see a button input "Cite Dataset"
+		When I press "Cite Dataset"
+		Then I should see "Text"
+		And I should see "RIS"
+		And I should see "BibTeX"
+		And I should see "EndNote"
+
+	@ok @javascript @pr521
+	Scenario: To get Text file after clicking the Text button in the citation box
+		Given I am not logged in to Gigadb web site
+		And I am on "/dataset/101001"
+		And I press "Cite Dataset"
+		When I click on the "Text" button
+		Then I go to "http://data.datacite.org/text/x-bibliography/10.5072/101001"
+		And I should see "The resource you are looking for doesn't exist."
```

Comment:
Hi @rija @pli888 
I found that the `full_doi` in local gigaDB  is `10.5072/dataset_id`, but in `gigadb.org` is `10.5524/dataset_id`. 
For the acceptance test, it has to use `10.5072/dataset_id` to pass. 
But in fact, the citation file will be downloaded only if the url is `http://data.datacite.org/text/x-bibliography/10.5524/101001`.  Why there is different `doi` for local gigadb and production gigadb?
And for this PR, what is the proper way to replace the  `10.5072/dataset_id ` with `10.5524/dataset_id` in order to get the data from datacite?

Reply:
A dev deployment of GigaDB used `10.5072/dataset_id` for DOIs because the `10.5072` prefix was used for test minting. However, DOI testing on DataCite has changed - see this [blog post](https://blog.datacite.org/test-prefix-10-5072-retiring-june-1/).

Reply:
I replaced the test prefix `10.5072` with gigascience prefix `10.5524`.